src/zui/ZUIText/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC } from 'react';
+import { Typography } from '@mui/material';
+
+type TextVariant =
+  | 'headingLg'
+  | 'headingMd'
+  | 'headingSm'
+  | 'bodyMdRegular'
+  | 'bodyMdBold'
+  | 'bodySmRegular'
+  | 'bodySmBold'
+  | 'linkMd'
+  | 'linkSm'
+  | 'labelMdMedium'
+  | 'labelMdRegular';
+
+interface ZUITextProps {
+  content: string;
+  color?: 'primary' | 'secondary';
+  component: 'a' | 'div' | 'p' | 'span';
+  variant: TextVariant;
+}
+
+const ZUIText: FC<ZUITextProps> = ({ component, color, content, variant }) => {
+  return (
+    <Typography color={color} component={component} variant={variant}>
```

Comment:
Instead of passing `variant` to `Typography` and rely on MUI theming, I think we should "hard-code" the various `ZUIText` styles here. I'm not sure `Typography` is the best component to use here at all if we don't need the `variant` / theme integration..